simulation/webots/controllers/follower_vehicle/Makefile

```diff
@@ -11,6 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
```

Comment:
Remove this extra line

Reply:
No. This extra line is to separate the license and the author. This one is necessary.